app/src/main/res/values/strings.xml

```diff
@@ -72,6 +72,9 @@
     <string name="save_image_as_previewed">Save image as previewed</string>
     <string name="save_as_image_preview_desc">Save output image as visible on preview (fixes inversion)</string>
 
+    <string name="save_video_as_previewed">Save video as previewed</string>
```

Comment:
To make the description better, we can possibly mention "(fixes inversion in front camera)" instead of just mentioning "(fixes inversion)" as the back camera is already saving the image/video as previewed

Reply:
Have fixed this in the latest amended commits. Thanks for the feedback!